### DISCUSSION TITLE
Timed quote test

### DISCUSSION THREAD

**Discussion by lukew3** [monkeytypegame/monkeytype#1367] [Ideas] [answered]
There needs to be a mode where you can type a quote in timed mode. I've noticed that most people's top criticism of Monkeytype is that the typing tests are just lists of words, not like actual sentences. If you could toggle between dictionary or quotes in timed mode, I think that this criticism would lose value. 
* The option for a quote would be in the row above the type of test selection beside the punctuation and numbers options. 
  * Punctuation and numbers options should grey out after the realistic mode is selected
  * Not sure if quote would be the right name for the mode; could be called something along the lines of realistic, practical, formal, etc.
* To allow longer timed tests, multiple quotes could be put one after the other.
* Could also have quotes in word count mode, but I think that it is more important to have it in timed mode

**Comment by refact0r** [comment, marked as answer]
> I've noticed that most people's top criticism of Monkeytype is that the typing tests are just lists of words, not like actual sentences.

Isn't that why quote mode exists? I don't see the point to having a timed quote mode. If you want to type real sentences, just type regular quotes.

**Reply by Miodec** [reply]
I agree

**Reply by swirajaya** [reply]
Currently there is no way to test my typing speed sustained over a longer period of time with real sentences. We're talking about e.g. 10 minutes or even indefinitely, where thicc quotes are not nearly enough. This is where the timed quote mode comes in.

Granted there's the custom mode where I can paste a text from somewhere to make my own typing test (or zen mode and copy the text from a book or something) but auto-generation of realistic text by stringing quotes together would be very nice.
